.github/workflows/e2e-test.yml

```diff
@@ -80,12 +80,6 @@ jobs:
           go install github.com/onsi/ginkgo/v2/ginkgo
           go get github.com/onsi/gomega/...
 
-      - name: Setup KinD
```

Comment:
Why are we deleting this?

Reply:
Because we have two "Setup KinD" in e2e-test.